### PR TITLE
Updated config.tf for Tectonic - 1.7.3 Release

### PR DESCRIPTION
**Pull request by sudhaponnaganti**

### CODE REVIEW

File: config.tf [hunk 1]
```diff
@@ -25,40 +25,40 @@ variable "tectonic_container_images" {
...
-    calico_cni                      = "quay.io/calico/cni:v1.9.1-4-g23fcd5f"
+    bootkube                        = "quay.io/coreos/bootkube:v0.6.0"
+    calico                          = "quay.io/calico/node:v2.4.1"
+    calico_cni                      = "quay.io/calico/cni:v1.9.1-16-gea55960"
     config_reload                   = "quay.io/coreos/configmap-reload:v0.0.1"
     container_linux_update_operator = "quay.io/coreos/container-linux-update-operator:v0.2.2"
     console                         = "quay.io/coreos/tectonic-console:v1.8.6"
     error_server                    = "quay.io/coreos/tectonic-error-server:1.0"
-    etcd                            = "quay.io/coreos/etcd:v3.1.8"
-    etcd_operator                   = "quay.io/coreos/etcd-operator:v0.4.2"
```
Comment (Quentin-M) [inline config.tf]: @xiang90 @hongchaodeng Can we update to etcd v3.1.8 and etcd-operator v0.4.2?

File: config.tf [hunk 2]
```diff
@@ -25,40 +25,40 @@ variable "tectonic_container_images" {
...
-    flannel                         = "quay.io/coreos/flannel:v0.8.0-amd64"
```
Comment (Quentin-M) [inline config.tf]: We should keep -amd64 I guess?

File: config.tf [hunk 3]
```diff
@@ -25,40 +25,40 @@ variable "tectonic_container_images" {
...
+    etcd                            = "quay.io/coreos/etcd:v3.2.5"
+    etcd_operator                   = "quay.io/coreos/etcd-operator:v0.5.0"
+    flannel                         = "quay.io/coreos/flannel:v0.8.0"
     flannel_cni                     = "quay.io/coreos/flannel-cni:0.1.0"
-    heapster                        = "gcr.io/google_containers/heapster:v1.4.0"
-    hyperkube                       = "quay.io/coreos/hyperkube:v1.7.2_coreos.0"
-    identity                        = "quay.io/coreos/dex:v2.5.0"
```
Comment (Quentin-M) [inline config.tf]: @rithujohn191 Is 2.6.0 okay for 1.7.3 or do we plan to get another version?

File: config.tf [hunk 4]
```diff
@@ -25,40 +25,40 @@ variable "tectonic_container_images" {
...
+    heapster                        = "gcr.io/google_containers/heapster:v1.4.1"
+    hyperkube                       = "quay.io/coreos/hyperkube:v1.7.3_coreos.0"
+    identity                        = "quay.io/coreos/dex:v2.6.0"
```
Comment (Quentin-M) [inline config.tf]: @rithujohn191 Is 2.6.0 okay for 1.7.3 or do we plan to get another version?

File: config.tf [hunk 5]
```diff
@@ -25,40 +25,40 @@ variable "tectonic_container_images" {
...
+    calico_cni                      = "quay.io/calico/cni:v1.9.1-16-gea55960"
     config_reload                   = "quay.io/coreos/configmap-reload:v0.0.1"
     container_linux_update_operator = "quay.io/coreos/container-linux-update-operator:v0.2.2"
     console                         = "quay.io/coreos/tectonic-console:v1.8.6"
     error_server                    = "quay.io/coreos/tectonic-error-server:1.0"
-    etcd                            = "quay.io/coreos/etcd:v3.1.8"
-    etcd_operator                   = "quay.io/coreos/etcd-operator:v0.4.2"
-    flannel                         = "quay.io/coreos/flannel:v0.8.0-amd64"
+    etcd                            = "quay.io/coreos/etcd:v3.2.5"
+    etcd_operator                   = "quay.io/coreos/etcd-operator:v0.5.0"
```
Comment (Quentin-M) [inline config.tf]: @xiang90 @hongchaodeng Can we update to etcd v3.1.8 and etcd-operator v0.4.2?

File: config.tf [hunk 6]
```diff
@@ -70,9 +70,9 @@ variable "tectonic_versions" {
...
   default = {
-    alertmanager  = "v0.7.1"
-    etcd          = "3.1.8"
-    kubernetes    = "1.7.1+tectonic.1"
+    alertmanager  = "v0.8.0"
+    etcd          = "3.2.5"
+    kubernetes    = "1.7.3+tectonic.1"
     monitoring    = "1.4.1"
     prometheus    = "v1.7.1"
     tectonic      = "1.7.1-tectonic.1"
```
Comment (squat) [inline config.tf]: we should also bump here

File: config.tf (hunk 4 above)
Comment (rithujohn191) [inline config.tf]: 2.6.0 is fine :+1:

File: config.tf (hunk 5 above)
Comment (sudhaponnaganti) [inline config.tf]: @xiang90 - can you review etcd version for 1.73, so we can move forward with the release milestone

**Comment by Quentin-M** [comment]
ok to test

**Comment by Quentin-M** [comment]
ok to test

**Comment by Quentin-M** [comment]
ok to test

**Comment by Quentin-M** [comment]
Rebased for conflict and triggering tests.

**Comment by Quentin-M** [comment]
```
        --- FAIL: Test/Cluster/NetworkPolicy (302.40s)
            --- FAIL: Test/Cluster/NetworkPolicy/DefaultDeny (250.88s)
            	cluster_test.go:457: error still reachable nginx-73s4d: error svc was reachable: ping job didn't succeed: timed out waiting for the condition
            	cluster_test.go:457: error still reachable nginx-73s4d: error svc was reachable: ping job didn't succeed: timed out waiting for the condition
            	cluster_test.go:462: default deny failed: timed out waiting for the condition
            --- PASS: Test/Cluster/NetworkPolicy/NetworkPolicy (15.47s)
```

@abhinavdahiya @squat do you know if this would be a known flake or due to the calico bump?

**Comment by abhinavdahiya** [comment]
@Quentin-M #1637 correctly bumps calico. Bump to calico need additions to rbac rules.

**Comment by Quentin-M** [comment]
@abhinavdahiya Thank you!

**Comment by amrutac** [comment]
This should include an update to console version as well.  Currently blocked by https://github.com/coreos/tectonic-installer/pull/1733.

**Comment by Quentin-M** [comment]
@amrutac Yeah, thanks for putting that in writings. @derekparker we still need to wait for a console bump before cutting KVO - I will let you know when we are all set!

**Comment by amrutac** [comment]
@Quentin-M: @derekparker: @rithujohn191 Console `v1.9.1` is ready.

**Comment by Quentin-M** [comment]
Thank you. I just rebased with Console 1.9.1.

File: config.tf [hunk 7]
```diff
@@ -25,40 +25,40 @@ variable "tectonic_container_images" {
...
-    pod_checkpointer                = "quay.io/coreos/pod-checkpointer:980d1b4b4b8374240c240fb0f85e3a8d9c51663c"
+    pod_checkpointer                = "quay.io/coreos/pod-checkpointer:3517908b1a1837e78cfd041a0e51e61c7835d85f"
     prometheus                      = "quay.io/prometheus/prometheus:v1.7.1"
     prometheus_config_reload        = "quay.io/coreos/prometheus-config-reloader:v0.0.2"
-    prometheus_operator             = "quay.io/coreos/prometheus-operator:v0.11.0"
+    prometheus_operator             = "quay.io/coreos/prometheus-operator:v0.11.1"
     stats_emitter                   = "quay.io/coreos/tectonic-stats:6e882361357fe4b773adbf279cddf48cb50164c1"
     stats_extender                  = "quay.io/coreos/tectonic-stats-extender:487b3da4e175da96dabfb44fba65cdb8b823db2e"
-    tectonic_channel_operator       = "quay.io/coreos/tectonic-channel-operator:0.4.1"
+    tectonic_channel_operator       = "quay.io/coreos/tectonic-channel-operator:0.4.2"
```
Comment (yifan-gu) [inline config.tf]: 0.4.4

File: config.tf [hunk 8]
```diff
@@ -25,40 +25,40 @@ variable "tectonic_container_images" {
...
+    identity                        = "quay.io/coreos/dex:v2.6.0"
     ingress_controller              = "gcr.io/google_containers/nginx-ingress-controller:0.9.0-beta.11"
     kenc                            = "quay.io/coreos/kenc:0.0.2"
     kubedns                         = "gcr.io/google_containers/k8s-dns-kube-dns-amd64:1.14.4"
     kubednsmasq                     = "gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64:1.14.4"
     kubedns_sidecar                 = "gcr.io/google_containers/k8s-dns-sidecar-amd64:1.14.4"
-    kube_state_metrics              = "quay.io/coreos/kube-state-metrics:v0.5.0"
+    kube_state_metrics              = "quay.io/coreos/kube-state-metrics:v1.0.0"
     kube_version                    = "quay.io/coreos/kube-version:0.1.0"
     kube_version_operator           = "quay.io/coreos/kube-version-operator:v1.7.1-kvo.6"
```
Comment (yifan-gu) [inline config.tf]: kvo is not bumped

File: config.tf (hunk 7 above)
Comment (Quentin-M) [inline config.tf]: sweet, thank you!

File: config.tf (hunk 8 above)
Comment (Quentin-M) [inline config.tf]: yes, it is not out yet (cc @derekparker).

File: config.tf (hunk 7 above)
Comment (yifan-gu) [inline config.tf]: actually 0.4.3

Comment (Quentin-M) [inline config.tf]: I reverted that change anyways

**Comment by Quentin-M** [comment]
retest this please

**Comment by Quentin-M** [comment]
retest this please

**Comment by Quentin-M** [comment]
retest this please

**Comment by Quentin-M** [comment]
Jenkins CI is dead for the most part.

**Comment by Quentin-M** [comment]
```
=== RUN   Test
=== RUN   Test/Common
=== RUN   Test/Common/APIAvailable
=== RUN   Test/Cluster
=== RUN   Test/Cluster/AllNodesRunning
=== RUN   Test/Cluster/GetIdentityLogs
=== RUN   Test/Cluster/AllPodsRunning
=== RUN   Test/Cluster/KillAPIServer
=== RUN   Test/Cluster/AllResourcesCreated
--- PASS: Test (140.66s)
    --- PASS: Test/Common (0.46s)
        --- PASS: Test/Common/APIAvailable (0.46s)
        	common_test.go:20: API server is available.
    --- PASS: Test/Cluster (140.20s)
        --- PASS: Test/Cluster/AllNodesRunning (0.19s)
        	cluster_test.go:119: node ip-10-0-19-174.ca-central-1.compute.internal ready
        	cluster_test.go:119: node ip-10-0-19-201.ca-central-1.compute.internal ready
        	cluster_test.go:119: node ip-10-0-3-186.ca-central-1.compute.internal ready
        	cluster_test.go:119: node ip-10-0-58-203.ca-central-1.compute.internal ready
        	cluster_test.go:146: Successfully found 4 ready nodes.
        --- PASS: Test/Cluster/GetIdentityLogs (0.43s)
        	cluster_test.go:165: Successfully gathered identity logs.
        --- PASS: Test/Cluster/AllPodsRunning (0.23s)
        	cluster_test.go:86: All pods are ready.
        --- PASS: Test/Cluster/KillAPIServer (126.59s)
        	smoke_test.go:118: failed with error: old API server kube-apiserver-fc9kc still running
        	smoke_test.go:119: retrying in 10s
        	smoke_test.go:118: failed with error: failed to get API server pod: Get https://qmachu-api.tectonic.dev.coreos.systems:443/api/v1/namespaces/kube-system/pods?labelSelector=k8s-app%3Dkube-apiserver: unexpected EOF
        	smoke_test.go:119: retrying in 10s
        	smoke_test.go:118: failed with error: API server has not yet been running for more than one check
        	smoke_test.go:119: retrying in 10s
        	cluster_test.go:269: API server pods successfully came back up.
        --- PASS: Test/Cluster/AllResourcesCreated (12.76s)
        	cluster_test.go:276: looking for resources defined by the provided manifests...
        	cluster_test.go:301: OK : [Kind: etcd.coreos.com/v1beta1:Cluster | Namespace: kube-system | Name: kube-etcd] - etcd/migrate-etcd-cluster.json
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: kube-system | Name: etcd-client-tls] - experimental/etcd-client-tls.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: kube-system | Name: etcd-operator] - experimental/etcd-operator.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: kube-system | Name: etcd-peer-tls] - experimental/etcd-peer-tls.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: kube-system | Name: etcd-server-tls] - experimental/etcd-server-tls.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: kube-system | Name: etcd-service] - experimental/etcd-service.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: kube-system | Name: kube-etcd-network-checkpointer] - experimental/kenc.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: kube-system | Name: kube-apiserver] - manifests/kube-apiserver-secret.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: kube-system | Name: kube-apiserver] - manifests/kube-apiserver.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: kube-system | Name: kube-cloud-cfg] - manifests/kube-cloud-config.yaml
        	cluster_test.go:301: OK : [Kind: policy/v1beta1:PodDisruptionBudget | Namespace: kube-system | Name: kube-controller-manager] - manifests/kube-controller-manager-disruption.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: kube-system | Name: kube-controller-manager] - manifests/kube-controller-manager-secret.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: kube-system | Name: kube-controller-manager] - manifests/kube-controller-manager.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: kube-system | Name: kube-dns] - manifests/kube-dns.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: kube-system | Name: kube-dns] - manifests/kube-dns.yaml
        	cluster_test.go:301: OK : [Kind: /v1:ConfigMap | Namespace: kube-system | Name: kube-flannel-cfg] - manifests/kube-flannel.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: kube-system | Name: kube-flannel] - manifests/kube-flannel.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: kube-system | Name: kube-proxy] - manifests/kube-proxy.yaml
        	cluster_test.go:301: OK : [Kind: policy/v1beta1:PodDisruptionBudget | Namespace: kube-system | Name: kube-scheduler] - manifests/kube-scheduler-disruption.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: kube-system | Name: kube-scheduler] - manifests/kube-scheduler.yaml
        	cluster_test.go:301: OK : [Kind: rbac.authorization.k8s.io/v1beta1:ClusterRoleBinding | Namespace:  | Name: system:default-sa] - manifests/kube-system-rbac-role-binding.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: kube-system | Name: pod-checkpointer] - manifests/pod-checkpointer.yaml
        	cluster_test.go:301: OK : [Kind: /v1:ConfigMap | Namespace: tectonic-system | Name: tectonic-config] - tectonic/config.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: tectonic-console] - tectonic/console/deployment.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: tectonic-console] - tectonic/console/service.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: etcd-operator] - tectonic/etcd/etcd-operator.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: kube-system | Name: heapster] - tectonic/heapster/deployment.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: kube-system | Name: heapster] - tectonic/heapster/service.yaml
        	cluster_test.go:301: OK : [Kind: /v1:ConfigMap | Namespace: tectonic-system | Name: tectonic-identity] - tectonic/identity/configmap.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: tectonic-identity] - tectonic/identity/deployment.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: tectonic-identity] - tectonic/identity/services.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: tectonic-identity-api] - tectonic/identity/services.yaml
        	cluster_test.go:301: OK : [Kind: /v1:ConfigMap | Namespace: tectonic-system | Name: tectonic-custom-error] - tectonic/ingress/default-backend/configmap.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: default-http-backend] - tectonic/ingress/default-backend/deployment.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: default-http-backend] - tectonic/ingress/default-backend/service.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: tectonic-lb] - tectonic/ingress/hostport/service.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:Ingress | Namespace: tectonic-system | Name: tectonic-ingress] - tectonic/ingress/ingress.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: tectonic-ingress-controller] - tectonic/ingress/nodeport/deployment.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: tectonic-lb] - tectonic/ingress/nodeport/service.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: tectonic-system | Name: alertmanager-main] - tectonic/monitoring/alertmanager-config.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: alertmanager-main] - tectonic/monitoring/alertmanager-service.yaml
        	cluster_test.go:301: OK : [Kind: monitoring.coreos.com/v1alpha1:Alertmanager | Namespace: tectonic-system | Name: main] - tectonic/monitoring/alertmanager.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: kube-system | Name: kube-controller-manager-prometheus-discovery] - tectonic/monitoring/kube-controller-manager-svc.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: kube-system | Name: kube-scheduler-prometheus-discovery] - tectonic/monitoring/kube-scheduler-svc.yaml
        	cluster_test.go:301: OK : [Kind: rbac.authorization.k8s.io/v1beta1:ClusterRoleBinding | Namespace:  | Name: kube-state-metrics] - tectonic/monitoring/kube-state-metrics-cluster-role-binding.yaml
        	cluster_test.go:301: OK : [Kind: rbac.authorization.k8s.io/v1beta1:ClusterRole | Namespace:  | Name: kube-state-metrics] - tectonic/monitoring/kube-state-metrics-cluster-role.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: kube-state-metrics] - tectonic/monitoring/kube-state-metrics-deployment.yaml
        	cluster_test.go:301: OK : [Kind: /v1:ServiceAccount | Namespace: tectonic-system | Name: kube-state-metrics] - tectonic/monitoring/kube-state-metrics-service-account.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: kube-state-metrics] - tectonic/monitoring/kube-state-metrics-service.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: node-exporter] - tectonic/monitoring/node-exporter-ds.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: node-exporter] - tectonic/monitoring/node-exporter-svc.yaml
        	cluster_test.go:301: OK : [Kind: rbac.authorization.k8s.io/v1beta1:ClusterRoleBinding | Namespace:  | Name: prometheus-k8s] - tectonic/monitoring/prometheus-k8s-cluster-role-binding.yaml
        	cluster_test.go:301: OK : [Kind: rbac.authorization.k8s.io/v1beta1:ClusterRole | Namespace:  | Name: prometheus-k8s] - tectonic/monitoring/prometheus-k8s-cluster-role.yaml
        	cluster_test.go:301: OK : [Kind: /v1:ConfigMap | Namespace: tectonic-system | Name: prometheus-k8s-rules] - tectonic/monitoring/prometheus-k8s-rules.yaml
        	cluster_test.go:301: OK : [Kind: /v1:ServiceAccount | Namespace: tectonic-system | Name: prometheus-k8s] - tectonic/monitoring/prometheus-k8s-service-account.yaml
        	cluster_test.go:301: OK : [Kind: monitoring.coreos.com/v1alpha1:ServiceMonitor | Namespace: tectonic-system | Name: alertmanager] - tectonic/monitoring/prometheus-k8s-service-monitor-alertmanager.yaml
        	cluster_test.go:301: OK : [Kind: monitoring.coreos.com/v1alpha1:ServiceMonitor | Namespace: tectonic-system | Name: kube-apiserver] - tectonic/monitoring/prometheus-k8s-service-monitor-apiserver.yaml
        	cluster_test.go:301: OK : [Kind: monitoring.coreos.com/v1alpha1:ServiceMonitor | Namespace: tectonic-system | Name: kube-controller-manager] - tectonic/monitoring/prometheus-k8s-service-monitor-kube-controller-manager.yaml
        	cluster_test.go:301: OK : [Kind: monitoring.coreos.com/v1alpha1:ServiceMonitor | Namespace: tectonic-system | Name: kube-scheduler] - tectonic/monitoring/prometheus-k8s-service-monitor-kube-scheduler.yaml
        	cluster_test.go:301: OK : [Kind: monitoring.coreos.com/v1alpha1:ServiceMonitor | Namespace: tectonic-system | Name: kube-state-metrics] - tectonic/monitoring/prometheus-k8s-service-monitor-kube-state-metrics.yaml
        	cluster_test.go:301: OK : [Kind: monitoring.coreos.com/v1alpha1:ServiceMonitor | Namespace: tectonic-system | Name: kubelet] - tectonic/monitoring/prometheus-k8s-service-monitor-kubelet.yaml
        	cluster_test.go:301: OK : [Kind: monitoring.coreos.com/v1alpha1:ServiceMonitor | Namespace: tectonic-system | Name: node-exporter] - tectonic/monitoring/prometheus-k8s-service-monitor-node-exporter.yaml
        	cluster_test.go:301: OK : [Kind: monitoring.coreos.com/v1alpha1:ServiceMonitor | Namespace: tectonic-system | Name: prometheus-operator] - tectonic/monitoring/prometheus-k8s-service-monitor-prometheus-operator.yaml
        	cluster_test.go:301: OK : [Kind: monitoring.coreos.com/v1alpha1:ServiceMonitor | Namespace: tectonic-system | Name: prometheus] - tectonic/monitoring/prometheus-k8s-service-monitor-prometheus.yaml
        	cluster_test.go:301: OK : [Kind: monitoring.coreos.com/v1alpha1:Prometheus | Namespace: tectonic-system | Name: k8s] - tectonic/monitoring/prometheus-k8s.yaml
        	cluster_test.go:301: OK : [Kind: rbac.authorization.k8s.io/v1beta1:ClusterRoleBinding | Namespace:  | Name: prometheus-operator] - tectonic/monitoring/prometheus-operator-cluster-role-binding.yaml
        	cluster_test.go:301: OK : [Kind: rbac.authorization.k8s.io/v1beta1:ClusterRole | Namespace:  | Name: prometheus-operator] - tectonic/monitoring/prometheus-operator-cluster-role.yaml
        	cluster_test.go:301: OK : [Kind: /v1:ServiceAccount | Namespace: tectonic-system | Name: prometheus-operator] - tectonic/monitoring/prometheus-operator-service-account.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: prometheus-operator] - tectonic/monitoring/prometheus-operator-svc.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: prometheus-operator] - tectonic/monitoring/prometheus-operator.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: prometheus] - tectonic/monitoring/prometheus-svc.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: tectonic-monitoring-auth-alertmanager] - tectonic/monitoring/tectonic-monitoring-auth-alertmanager-deployment.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: tectonic-monitoring-auth-alertmanager] - tectonic/monitoring/tectonic-monitoring-auth-alertmanager-svc.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: tectonic-monitoring-auth-prometheus] - tectonic/monitoring/tectonic-monitoring-auth-prometheus-deployment.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Service | Namespace: tectonic-system | Name: tectonic-monitoring-auth-prometheus] - tectonic/monitoring/tectonic-monitoring-auth-prometheus-svc.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: tectonic-system | Name: tectonic-monitoring-auth] - tectonic/monitoring/tectonic-monitoring-auth-secret.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:Ingress | Namespace: tectonic-system | Name: tectonic-monitoring-ingress] - tectonic/monitoring/tectonic-monitoring-ingress.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Namespace | Namespace:  | Name: tectonic-system] - tectonic/namespace.yaml
        	cluster_test.go:301: OK : [Kind: rbac.authorization.k8s.io/v1beta1:ClusterRoleBinding | Namespace:  | Name: admin-user] - tectonic/rbac/binding-admin.yaml
        	cluster_test.go:301: OK : [Kind: rbac.authorization.k8s.io/v1beta1:ClusterRoleBinding | Namespace:  | Name: discovery] - tectonic/rbac/binding-discovery.yaml
        	cluster_test.go:301: OK : [Kind: rbac.authorization.k8s.io/v1beta1:ClusterRole | Namespace:  | Name: admin] - tectonic/rbac/role-admin.yaml
        	cluster_test.go:301: OK : [Kind: rbac.authorization.k8s.io/v1beta1:ClusterRole | Namespace:  | Name: user] - tectonic/rbac/role-user.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: tectonic-system | Name: tectonic-ca-cert-secret] - tectonic/secrets/ca-cert.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: tectonic-system | Name: tectonic-identity-grpc-client-secret] - tectonic/secrets/identity-grpc-client.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: tectonic-system | Name: tectonic-identity-grpc-server-secret] - tectonic/secrets/identity-grpc-server.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: tectonic-system | Name: tectonic-ingress-tls-secret] - tectonic/secrets/ingress-tls.yaml
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: tectonic-system | Name: tectonic-license-secret] - tectonic/secrets/license.json
        	cluster_test.go:301: OK : [Kind: /v1:Secret | Namespace: tectonic-system | Name: coreos-pull-secret] - tectonic/secrets/pull.json
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: tectonic-stats-emitter] - tectonic/stats-emitter.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:ThirdPartyResource | Namespace:  | Name: app-version.coreos.com] - tectonic/updater/app-version-kind.yaml
        	cluster_test.go:301: OK : [Kind: coreos.com/v1:AppVersion | Namespace: tectonic-system | Name: kubernetes] - tectonic/updater/app_versions/app-version-kubernetes.yaml
        	cluster_test.go:301: OK : [Kind: coreos.com/v1:AppVersion | Namespace: tectonic-system | Name: tectonic-cluster] - tectonic/updater/app_versions/app-version-tectonic-cluster.yaml
        	cluster_test.go:301: OK : [Kind: coreos.com/v1:AppVersion | Namespace: tectonic-system | Name: tectonic-monitoring] - tectonic/updater/app_versions/app-version-tectonic-monitoring.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:ThirdPartyResource | Namespace:  | Name: migration-status.coreos.com] - tectonic/updater/migration-status-kind.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: node-agent] - tectonic/updater/node-agent.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: container-linux-update-operator] - tectonic/updater/operators/container-linux-update-operator.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: kube-version-operator] - tectonic/updater/operators/kube-version-operator.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: tectonic-channel-operator] - tectonic/updater/operators/tectonic-channel-operator.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:DeploymentOrDaemonSet | Namespace: tectonic-system | Name: tectonic-prometheus-operator] - tectonic/updater/operators/tectonic-prometheus-operator.yaml
        	cluster_test.go:301: OK : [Kind: coreos.com/v1:ChannelOperatorConfig | Namespace: tectonic-system | Name: default] - tectonic/updater/tectonic-channel-operator-config.yaml
        	cluster_test.go:301: OK : [Kind: extensions/v1beta1:ThirdPartyResource | Namespace:  | Name: channel-operator-config.coreos.com] - tectonic/updater/tectonic-channel-operator-kind.yaml
        	cluster_test.go:301: OK : [Kind: /v1:ConfigMap | Namespace: tectonic-system | Name: tectonic-monitoring] - tectonic/updater/tectonic-monitoring-config.yaml
        	cluster_test.go:337: all resources defined by the provided manifests are present
PASS
```

**Pull request merged**